sql-join/where-are-they.sql

```diff
@@ -0,0 +1,4 @@
+select "addresses"."line1", "cities"."name" as "city", "addresses"."district", "countries"."name" as "country"
+from "addresses"
+join "cities" using ("cityId")
+join "countries" on "cities"."countryId" = "countries"."countryId";
```

Comment:
Same question here. It is odd seeing both styles used when just one style would work.

No need to change this! I'm mostly just curious.